athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/metadata/ListTablesRequest.java

```diff
@@ -74,6 +119,7 @@ public String toString()
         return "ListTablesRequest{" +
                 "queryId=" + getQueryId() +
                 ", schemaName='" + schemaName + '\'' +
+                ", pageSize=" + pageSize +
```

Comment:
next token?

Reply:
Why would we want to print nextToken? 
1. You wanted it to be encrypted by the connectors which means it's sensitive info.
2. It's encrypted so the value is meaningless printed out.

Reply:
Added